testsuite/draw_string/run.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+command += testshade ("-g 512 512 -od uint8 -o Cout out.tif test")
+outputs = [ "out.txt", "out.tif" ]
+# expect a few LSB failures
+failthresh = 0.004
+failpercent = 0.05
```

Comment:
A few tests need slightly higher fail thresholds, but if this passes without these two lines, please take them out.